src/PanZoom.js

```diff
@@ -671,7 +683,33 @@ class PanZoom extends React.Component<Props> {
   }
 
   render() {
-    const { children, style, disabled, disableKeyInteraction } = this.props
+    const {
+      children,
+      autoCenter,
+      autoCenterZoomLevel,
+      zoomSpeed,
+      doubleZoomSpeed,
+      disabled,
+      disableKeyInteraction,
+      realPinch,
+      keyMapping,
+      minZoom,
+      maxZoom,
+      enableBoundingBox,
+      boundaryRatioVertical,
+      boundaryRatioHorizontal,
+      noStateUpdate,
+      onPanStart,
+      onPan,
+      onPanEnd,
+      preventPan,
+      style,
+      onDoubleClick,
+      onMouseDown,
+      onKeyDown,
+      onTouchStart,
+      ...props
```

Comment:
Can we use another naming than `props`, it becomes a bit confusing between `this.props` and `props`. Maybe something like `...rest` or `...otherProps`